content/docs/csidriver/installation/operator/powerflex.md

```diff
@@ -43,7 +43,7 @@ Kubernetes Operators make it easy to deploy and manage the entire lifecycle of c
   - Optionally, enable sdc monitor by uncommenting the section for sidecar in manifest yaml. Please note the following: 
     - **If using sidecar**, you will need to edit the value fields under the HOST_PID and MDM fields by filling the empty quotes with host PID and the MDM IPs. 
     - **If not using sidecar**, please leave this commented out -- otherwise, the empty fields will cause errors.
-##### Example CR:  [config/samples/vxflex_v250_ops_411.yaml](https://github.com/dell/dell-csi-operator/blob/master/samples/vxflex_v250_ops_411.yaml)
+##### Example CR:  [config/samples/vxflex_v260_ops_411.yaml](https://github.com/dell/dell-csi-operator/blob/master/samples/vxflex_v260_ops_411.yaml)
```

Comment:
this link works, but we can change master -> main branch in url.

Reply:
updated